flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinatorTest.java

```diff
@@ -674,7 +674,14 @@ public void testIOExceptionForPeriodicSchedulingWithInactiveTasks() throws Excep
     /** Tests that do not trigger checkpoint when IOException occurred. */
     @Test
     public void testTriggerCheckpointAfterIOException() throws Exception {
-        testTriggerCheckpoint(new TestingIOExceptionCheckpointIDCounter(), IO_EXCEPTION);
```

Comment:
Wouldn't it be better to replace `TestingIOExceptionCheckpointIDCounter` with `IOExceptionCheckpointStorage` everywhere?